### ISSUE TITLE
When the data has negative values, the data and grid lines on the Y-axis are not displayed

### ISSUE THREAD

**Issue by zhangyu-1993** [syncfusion/flutter-widgets#515]
When the data has negative values, the data and grid lines on the Y-axis are not displayed
The version used is the latest 19.4.3
<img width="314" alt="截屏2021-12-28 下午3 36 54" src="https://user-images.githubusercontent.com/45927123/147541199-def24c01-50b6-45be-974d-78333b305fc3.png">

**Comment by Yuvaraj-Gajaraj** [comment]
Hi @zhangyu-1993,

Greetings from Syncfusion. We have analyzed your query and with the latest version of the chart, with the negative values, the axis labels are visible properly. And from the provided image, we found that you have a dark background for the chart, and if you have set the color of axis labels, gridline color also in dark for example in black, the axis labels, gridline might not be visible. We are not aware of the exact scenario of the cause, so try to replicate the scenario with the attached sample or get back to us with your code and replication steps. This will be more helpful for us to provide a solution sooner.

Sample: [github_515.zip](https://github.com/syncfusion/flutter-widgets/files/7789157/github_515.zip)

Regards,
Yuvaraj.
